guides/common/modules/con_using-pxe-to-provision-hosts.adoc

```diff
@@ -36,18 +36,25 @@ endif::[]
 With {ProjectName}, you can perform both BIOS and UEFI based PXE provisioning.
 Both BIOS and UEFI interfaces work as interpreters between the computer's operating system and firmware, initializing the hardware components and starting the operating system at boot time.
 
-ifdef::satellite[]
-For information about supported workflows, see https://access.redhat.com/solutions/2674001[Supported architectures and provisioning scenarios].
-endif::[]
-
+.PXE loaders
 In {Project} provisioning, the PXE loader option defines the DHCP `filename` option to use during provisioning.
 For BIOS systems, use the *PXELinux BIOS* option to enable a provisioned node to download the `pxelinux.0` file over TFTP.
 For UEFI systems, use the *PXEGrub2 UEFI* option to enable a TFTP client to download `grub2/grubx64.efi` file, or use the *PXEGrub2 UEFI HTTP* option to enable an UEFI HTTP client to download `grubx64.efi` from {SmartProxy} with the HTTP Boot feature.
-ifndef::satellite[]
-Use SecureBoot options to enable a client to download the `shim.efi` bootstrap bootloader that then loads the signed `grubx64.efi`.
-Other PXE loaders like PXELinux UEFI, Grub2 ELF or iPXE Chain, require additional configuration. These workflows are not documented at the moment.
+
+{ProjectName} supports UEFI Secure Boot.
+SecureBoot PXE loaders enable a client to download the `shim.efi` bootstrap bootloader that then loads the signed `grubx64.efi`.
+Use the *Grub2 UEFI SecureBoot* PXE loader for PXE-boot provisioning or *Grub2 UEFI HTTPS SecureBoot* for HTTP-boot provisioning.
+You can only provision operating systems that support Secure Boot, such as {EL} 7 or later.
```

Comment:
That’s not entirely accurate. If my Smart Proxy runs on RHEL and I try to provision a CentOS machine, it may not work due to differences in bootloader binaries. Additionally, provisioning a RHEL machine with a different version may also fail if there have been updates in the Secure Boot configurations between versions.

Reply:
Oh, right, that was the limitation to hosts that run the same OS version as the Smart Proxy! Gotcha.